metron-interface/metron-config/src/app/shared/metron-alerts.ts

```diff
@@ -15,9 +17,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@Injectable()
 export class MetronAlerts {
 
-  public static SUCESS_MESSAGE_DISPALY_TIME = 5000;
+  public static SUCCESS_MESSAGE_DISPALY_TIME = 5000;
```

Comment:
This is still misspelled.  I think it should be `SUCCESS_MESSAGE_DISPLAY_TIME`?

Reply:
Fixed. Thx